android/titanium/src/java/org/appcelerator/titanium/proxy/TiViewProxy.java

```diff
@@ -1166,6 +1172,46 @@ public TiViewProxy getParent()
 		return this.parent.get();
 	}
 
+	// clang-format off
+	@Kroll.method
+	@Kroll.getProperty
+	public String getBackgroundDisabledColor()
+	// clang-format on
+	{
+		// Try to get the background drawable if one is available.
+		TiBackgroundDrawable backgroundDrawable = getOrCreateView().getBackground();
+		// Guard for views without color state backgrounds.
+		if (backgroundDrawable == null) {
+			return null;
+		} else {
+			try {
+				// Get the backgroundDrawable background as a StateListDrawable.
+				StateListDrawable stateListDrawable = ((StateListDrawable) backgroundDrawable.getBackground());
+				// Get the reflection methods.
+				Method getStateDrawableIndexMethod =
+					StateListDrawable.class.getMethod("getStateDrawableIndex", int[].class);
+				Method getStateDrawableMethod = StateListDrawable.class.getMethod("getStateDrawable", int.class);
+				// Get the disabled state's (as defined in TiUIHelper) index.
+				int index =
+					(int) getStateDrawableIndexMethod.invoke(stateListDrawable, TiUIHelper.BACKGROUND_DISABLED_STATE);
+				// Get the drawable at the index.
+				Drawable drawable = (Drawable) getStateDrawableMethod.invoke(stateListDrawable, index);
+				// Parse it as a ColorDrawable.
+				ColorDrawable colorDrawable = ((ColorDrawable) ((LayerDrawable) drawable).getDrawable(0));
+				// Transcript the color int to HexString.
+				String strColor = String.format("#%06X", 0xFFFFFF & colorDrawable.getColor());
```

Comment:
Could this return a color with an alpha channel?

Reply:
I have missed to check that. It can, so I changed the resulting String format.